### ISSUE TITLE
-addresstype=legacy argument not work for createmultisig.

### ISSUE THREAD

**Issue by passionofvc** [litecoin-project/litecoin#467]
-addresstype=legacy not work for createmultisig rpc,
it even return new sigwit address prefix with [M].
with bitcoin-0.16.0 it gave me correct legacy address prefix with [3].
with  litecoin-v0.16.0rc1 it not work.

according to 
https://blog.litecoin.org/litecoin-core-v0-16-0-release-candidate-e1ac751d7f33

> A new -addresstype argument has been added, which supports legacy, p2sh-segwit (default), and bech32 addresses. It controls what kind of addresses are produced by getnewaddress, getaccountaddress, and createmultisigaddress. A -changetype argument has also been added, with the same options, and by default equal to -addresstype, to control which kind of change is used

```
./litecoind --version
Litecoin Core Daemon version v0.16.0rc1
Copyright (C) 2011-2018 The Litecoin Core developers
Copyright (C) 2009-2018 The Bitcoin Core developers

./litecoind -addresstype=legacy -deprecatedrpc=createmultisig

```

$ bash cli.bash  validateaddress LY8tRVxi3WTy7ZeduHxUvP967jRcXihaGZ
{
  "isvalid": true,
  "address": "LY8tRVxi3WTy7ZeduHxUvP967jRcXihaGZ",
  "scriptPubKey": "76a9148da4599a647b65829997abe203f2aaa5a007611288ac",
  "ismine": true,
  "iswatchonly": false,
  "isscript": false,
  "iswitness": false,
  "pubkey": "0398ffc9d0efb9680e216dcd9bd3eaa85142965efc085407c903a669b018ccae26",
  "iscompressed": true,
  "account": "",
  "timestamp": 1523540692,
  "hdkeypath": "m/0'/0'/8'",
  "hdmasterkeyid": "67c5df8200c607b6e99bd1ab9fe0159fa5fbbf64"
}
[yah cli.bash  getnewaddressteaddress LY8tRVxi3WTy7ZeduHxUvP967jRcXihaGZ
LV1weTvCmSAkm76QMJ659fgHzeFo6ph7N8
$ bash cli.bash  validateaddress LV1weTvCmSAkm76QMJ659fgHzeFo6ph7N8
{
  "isvalid": true,
  "address": "LV1weTvCmSAkm76QMJ659fgHzeFo6ph7N8",
  "scriptPubKey": "76a9146b6bc075b483e78afa3271a24aa3eda60405a27688ac",
  "ismine": true,
  "iswatchonly": false,
  "isscript": false,
  "iswitness": false,
  "pubkey": "032d14f8140af74d0ae1eca4a12486cf3a610037548025ab68922e86c1fb2028ed",
  "iscompressed": true,
  "account": "",
  "timestamp": 1523540692,
  "hdkeypath": "m/0'/0'/9'",
  "hdmasterkeyid": "67c5df8200c607b6e99bd1ab9fe0159fa5fbbf64"
}
$ bash cli.bash  createmultisig 2 [\"0398ffc9d0efb9680e216dcd9bd3eaa85142965efc085407c903a669b018ccae26\",\"032d14f8140af74d0ae1eca4a12486cf3a610037548025ab68922e86c1fb2028ed\"]
{
  "address": "MWN47wCy3tiKW8C5HcD3PCRDs8hR2dnTLf",
  "redeemScript": "52210398ffc9d0efb9680e216dcd9bd3eaa85142965efc085407c903a669b018ccae2621032d14f8140af74d0ae1eca4a12486cf3a610037548025ab68922e86c1fb2028ed52ae"
}

**Comment by passionofvc** [comment]
in https://github.com/bitcoin/bitcoin/pull/13072/files
does litecoind need to include this feture?
need add address_type to createmultisig .
`createmultisig nrequired [\"key\",...] ( \"address_type\" )
`
```

    // Get the output type
    OutputType output_type = OutputType::LEGACY;
    if (!request.params[2].isNull()) {
        output_type = ParseOutputType(request.params[2].get_str(), output_type);
        if (output_type == OutputType::NONE) {
            throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, strprintf("Unknown address type '%s'", request.params[2].get_str()));
        }
    }
    
    const CScript inner = CreateMultisigRedeemscript(required, pubkeys);
    CBasicKeyStore keystore;
    const CTxDestination dest = AddAndGetDestinationForScript(keystore, inner, output_type);
```

**Comment by thrasher-** [comment]
This was fixed in v0.17.1
